migrations/20230419161219_remove-inactive-friendships-without-history.up.sql

```diff
@@ -0,0 +1,11 @@
+DELETE FROM friendships WHERE id IN(
+  SELECT f.id 
+	FROM friendships f
+	LEFT JOIN friendship_history h
```

Comment:
In this particular case I'd use Inner join

Reply:
I need a left join just to check that the friendship exists but there is no row in the friendship_history reference to it